.github/workflows/functests.yml

```diff
@@ -76,11 +76,11 @@ jobs:
 
       - name: Run migration tests with DirectCSI v3.2.2
         run: |
-          sudo rm -fr /var/lib/directpv
+          sudo rm -fr /var/lib/directpv/mnt
```

Comment:
`sudo rm -fr /var/lib/directpv` will fail because the we force delete the pods and the container paths would still remain mounted to `/var/lib/directpv/tmpfs` and CSI will lazily umount these container paths by calling `NodeUnpublishVolume` lazily. So, `sudo rm -fr /var/lib/directpv` will throw EBUSY. So, we remove the exact sub directory - `/mnt` here

Also, we don't create any other sub-dirs inside `/var/lib/directpv/` other than `/mnt` so far. even on older versions. So, it should be fine for upgrade tests.

Reply:
Ignore this, this is because we were removing before umounting tmpfs.

thanks @balamurugana for pointing this out.